frontend/src/modules/app/ToolBar.tsx

```diff
@@ -161,16 +189,33 @@ export const ToolBar = ({
     string,
     Range
   >;
+  /**
+   * State to manage toggling ventilationState
+   */
   const [isVentilatorOn, setIsVentilatorOn] = React.useState(ventilating);
+  /**
+   * State to manage label for Landing page
+   */
   const [landingLabel, setLandingLabel] = useState('Loading...');
+  /**
+   * State to manage ventilation label
+   * Label is Dynamic based on ventilation state
+   */
   const [label, setLabel] = useState('Start Ventilation');
+  /**
+   * State to toggle if Ventilating isDisabled
```

Comment:
What's the relationship between the `isDisabled` state and the `isVentilatorOn` state? What would it mean for `ventilating` to be `isDisabled` and `isVentilatorOn` to be true? Why do we need to have both `isDisabled` and `isVentilatorOn`? (i.e. what is each state needed for?)

Reply:
As per above comments `isVentilatorOn` is not is use anymore and should be deprecated once removed references from UI.

We dont have any relation between `isVentilatorOn` & `isDisabled`. While `ventilating` is waiting to happen on backend, `isDisabled` is true so that user is not allowed to click the Start/Pause Ventilating button.